spec/controllers/mfa_confirmation_controller_spec.rb

```diff
@@ -96,6 +96,26 @@
     end
   end
 
+  describe '#show_skip_additonal_mfa_link?' do
```

Comment:
* Minor: Typo on "additonal" (should be "additional")
* Since this is implemented in the concern, can we create a concern spec file with one set of tests rather than implementing the tests across all controllers which use the concern?